Jenkinsfile.rdgo

```diff
@@ -13,6 +13,7 @@ node(env.NODE) {
                 cp RPM-GPG-* /etc/pki/rpm-gpg/
                 dnf copr -y enable walters/buildtools-fedora
                 dnf install -y rpmdistro-gitoverlay
+                pip3 install awscli==1.15.20
```

Comment:
FWIW this also exists as `awscli` as an RPM.

Reply:
Ahh, nice. I had checked `python-awscli` and `python3-awscli`, but not `awscli` directly.